docker/scripts/entrypoint.sh

```diff
@@ -24,6 +24,8 @@
 
 set -eEuo pipefail
 
+RUN_USER="${RUN_USER:-}"
```

Comment:
Maybe fail early here if unset or empty. Otherwise we'd only fail later with `unbound variable`.

Reply:
Sounds good.